monitoring/rid_qualifier/host/auth.py

```diff
@@ -0,0 +1,8 @@
+from monitoring.monitorlib import auth_validation
+from monitoring.rid_qualifier.host import webapp
+from . import config
+
+
+requires_scope = auth_validation.requires_scope_decorator(
```

Comment:
We won't be validating authorization like this for this tool, so I think this file can be deleted.